### PR TITLE
feat: adds documentation site docs.api.puro.earth for monitoring

### PR DESCRIPTION
**Pull request by Addono**

### CODE REVIEW

File: .github/workflows/graphs.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v2.3.3
```
Comment (mathewrapid) [inline .github/workflows/graphs.yml:23]: Why is the version downgraded to v2.3.3?

Comment (mathewrapid) [reply .github/workflows/graphs.yml:23]: some upptime-bot magic

Comment (Addono) [reply .github/workflows/graphs.yml:23]: We can also just cherry-pick the change in.

Comment (Addono) [reply .github/workflows/graphs.yml:23]: It wasn't me 😇

**Pull request merged**